lib/io/stream/writable-object.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Interface for core functionality of write streams.
+ */
+export interface WritableObject {
+    write(chunk: string);
+    close();
```

Comment:
There is close function, I would add information who is responsible for calling that. Also it should return a promise - imagine a situation when the content is cached in memory and on close it need to be written into a file.

Reply:
This should be a general interface, and I think the calling convention is situation-specific. However, I only need the write method probably. So I will remove it.